libs/sr/consts/src/common.ts

```diff
@@ -29,13 +29,13 @@ export const allStatKeys = [
 export type StatKey = (typeof allStatKeys)[number]
 
 export const allElementalDamageKeys = [
-  'physical_dmg_',
-  'fire_dmg_',
-  'ice_dmg_',
-  'wind_dmg_',
-  'lightning_dmg_',
-  'quantum_dmg_',
-  'imaginary_dmg_',
```

Comment:
This is a small nitpick overall (and I'm fairly certain changing these probably is why the somnia check failed as of this commit), but I'm curious if there's any reason to keep the `_dmg_` string portion in this list of keys? 

For example, GO currently uses the "shorthand element" format (the change being made here), instead parsing the "full element" strings and converting them to the "shorthand" variant before doing XYZ with them.

The full version obviously has the advantage of requiring zero string preprocessing, but I suppose the shorthand is consistent with GO code (could be easier to port code with minimal changes as we keep moving things to `libs/common`?) It may be clearer to read on a glance? Perhaps easier to write in certain places? Probably negligible returns on the latter two fronts.

Reply:
It would probably be better to make an allElementalTypeKeys instead of changing this. An equivalent might already exist somewhere

Reply:
https://github.com/frzyc/genshin-optimizer/blob/7b45e410b0a59120b09e89a8803d6911f6909aca/libs/sr/consts/src/character.ts#L63

Reply:
The casing here is capitalized for whatever reason. Id have to look and see if there's a reason. I know it causes some issues in pando being capitalized so I could very easily be convinced to uncapitalize it

Reply:
the `_dmg_` convention is likely more historical and being more readable/explicit, I can be convinced to change it to tbe `wind_` variant for SRO, since there are no established standards, but it would require a database migration at the very minimum, which would probably be pretty painful.

Reply:
no db migration needed, no one is using it anyways

Reply:
what about scanners? the `_dmg_` keys are used by scanners as well, no?

Reply:
Only Kelz scanner supports SROD, and we can just ask him to update

Reply:
alright, for now I can change these to use the `ElementalTypeKeys`. I was confused as to the purpose of the `ElementalDamage` keys, when I totally forgot about the `-Type` keys that Van linked (oops). 

@nguyentvan7 should I make the `ElementalType` keys lowercase? I believe the sheets are printing both elemental type and path as uppercase, while ability (basic/skill/ult/etc) and dmg types (ST, AoE, bounce, support, etc) are all in lowercase. I'm not too sure how the executors work, and you're also more familiar with pando, so right now I'm erring towards caution and thinking of changing the switch case checks in icons.

Reply:
Yea there will be some things that need changing for the casing, I can address in a followup pr. Probably leave the upper case casing for now and I can make sure to update your staticon file when I get around to it